src/aioquic/quic/logger.py

```diff
@@ -32,6 +32,20 @@
 def hexdump(data: bytes) -> str:
     return binascii.hexlify(data).decode("ascii")
 
+def is_builtin_class(Object):
+    return Object.__class__.__module__ == 'builtins'
+
+def get_dataclass_attr(Object) -> dict:
```

Comment:
Why is this needed?

Reply:
It was needed for a simpler logging of the BBR/BBRv2 congestion control, where many attributes were present, it was thus better to have some method that automatically gets all its attributes.

This won't be needed for cubic, so I'll simply remove it for now